### DISCUSSION TITLE
What GEDCOM event tag should I use for a Nun taking vows?

### DISCUSSION THREAD

**Discussion by nk9** [FamilySearch/GEDCOM#405] [Q&A] [answered]
GEDCOM declares an Ordination person event tag (`ORDN`). This is described in [the standard documentation](https://gedcom.io/specifications/gedcom7-rc.pdf) as "Receiving authority to act in religious matters."

However, [it appears that Nuns aren't members of the clergy](https://uscatholic.org/articles/201403/debunking-five-myths-about-nuns/), they are technically laypeople. And thus, they aren't ordained, they take vows. A small number of nuns in certain religious traditions *have* been ordained at points, but it's far from normal.

So I'm wondering: what GEDCOM person event type should I use for the date that a Sister takes her vows? Do I need to just use a generic `RELI` event? And why isn't there a `VOWS` event, anyway?

I see that the [GEDCOM 5.3 standard document](https://gedcom.io/specifications/Gedcom5.3.pdf) contains a proposal for a `_VOW` event tag in Appendix B, but this seems to have disappeared from the version 7 release candidate (linked above). 🤔

**Comment by tychonievich** [comment, marked as answer]
`_VOW` was considered in 5.3 but not included in 5.4 two years later; that was before any of the current steering committee were involved in the discussions so I can't speak more to the rationale for that noninclusion.

I expect you'll find a mix of people using `ORDN` and people using `EVEN` for a nun taking vows, depending on their understanding of what those vows mean. Both can be clarified to mean a nun's vows with a `TYPE` substructure and have their religious order indicated with a `RELI` substructure.

If you expect to have a large number of such non-ordination vows, using an extension like `_VOW` may make sense. Extension tags beginning with `_` have always been allowed in all versions of GEDCOM. If you'd like to facilitate their interoperability you could register it on the <https://github.com/FamilySearch/GEDCOM-registries> repository. You could also request that it be considered for standardization with an pull request adding it to the [attribue-event-request](https://github.com/FamilySearch/GEDCOM/tree/main/attribute-event-requests) page or with an issue.

**Reply by nk9** [reply]
Thanks for the comprehensive response!
